apps/daimo-clippy/src/handlers.ts

```diff
@@ -146,29 +146,54 @@ async function getTokenList(): Promise<TokenList> {
   return tokenListPromise;
 }
 
-async function getUniswapRoute(kwargs: Map<string, string>): Promise<string> {
-  const strN = kwargs.get("num");
-  const strToken = kwargs.get("token");
-  if (!strN || !strToken) return "Must specify num and token";
+// Gets a swap quote from the onchain contract.
+async function getSwapQuote(kwargs: Map<string, string>): Promise<string> {
+  const strN = kwargs.get("fromAmount");
+  const strFromToken = kwargs.get("fromToken");
+  const strToToken = kwargs.get("toToken");
+  const strDaimoChainId = kwargs.get("daimoChainId");
+  if (!strN || !strFromToken || !strToToken || !strDaimoChainId)
+    return "Must specify fromAmount, fromToken and toToken";
```

Comment:
dont need kwArg fro daimoChainId, can use the one from env

Reply:
copy